package-lock.json

```diff
@@ -1369,26 +1369,26 @@
       }
     },
     "@eslint/eslintrc": {
-      "version": "1.0.4",
```

Comment:
should we update to this one? rather than downgrade?

Reply:
We had to downgrade to match another dependency

Reply:
@rainswerld can you approve if all is well?